CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 ## Unreleased
 
 FEATURES
+* modules/gateway-task: Add `health-sync` to `gateway-task` when ACLs are enabled.
+  When ACLs are enabled the gateway task does a `consul login` to get a token using the
+  IAM Auth Method. The ACL controller will delete the tokens during its reconciliation
+  cycle but the gateway task should do a `consul logout` when it stops to destroy
+  its tokens. The `health-sync` container automatically attempts a `consul logout`
+  when the task stops.
+  [[GH-120]](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/120)
```

Comment:
I think we could condense the changelog down to have a single higher-level bullet point about mesh-gateways, since mesh-gateway is newly added after the beta release. What do you think?